user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently Asked Questions
+
+## Can I Swarm X/Y/Z?
```

Comment:
Do we have to write Swarm with a capital?
(Same further down for "Swarmed" and "when Swarming an element")

Reply:
The feature "Swarming" gets a capital, but if you're referring to the action, no capital should be used.

Reply:
Done